console/src/main/scripts/plugins/metrics/less/metrics.less

```diff
@@ -2458,6 +2469,7 @@ section {
 
   padding-bottom: @grid-gutter-width/2;
   padding-top: @grid-gutter-width/2;
+  height: 120px;
```

Comment:
Well to be honest I did not tested it on different resolutions. I just didn't liked the way it snapped to new line, it was some kind of hot fix. So please @cardosogabriel if it is bad thing just give me a tip what is better, thanks.